src/functionalTest/java/uk/gov/hmcts/reform/sscs/functional/IssueFurtherEvidenceHandlerFunctionalTest.java

```diff
@@ -62,32 +78,35 @@ private void verifyEvidenceIssued() {
         Assertions.assertThat(docs)
             .extracting(SscsDocument::getValue)
             .extracting(SscsDocumentDetails::getEvidenceIssued)
-            .contains("Yes", "Yes", "Yes");
+            .filteredOn("Yes"::equalsIgnoreCase)
+            .hasSize(3);
     }
 
-    private void verifyEvidenceIssuedAndReasonableAdjustmentRaised() {
-        SscsCaseDetails caseDetails = findCaseById(ccdCaseId);
+    private void verifyEvidenceIssuedAndReasonableAdjustmentRaised(SscsCaseDetails caseDetailsIn) {
+        SscsCaseDetails caseDetails = findCaseById(String.valueOf(caseDetailsIn.getId()));
         SscsCaseData caseData = caseDetails.getData();
         List<SscsDocument> docs = caseData.getSscsDocument();
 
         Assertions.assertThat(docs)
             .extracting(SscsDocument::getValue)
             .extracting(SscsDocumentDetails::getEvidenceIssued)
-            .contains("Yes", "Yes", "Yes");
+            .filteredOn("Yes"::equalsIgnoreCase)
+            .hasSize(3);
```

Comment:
This is a fix on the test, which was incorrect before.

The test data actually returns 4 entries and we want 3 of them to contain "Yes". The previous syntax, according to documentation, was not guaranteeing that condition and was misleading in the sense that it looked as if we expect to have 3 entries, which is not the case and in the past has already led to discussion as to the opportunity to add `hasSize` checks (see https://github.com/hmcts/sscs-evidence-share/pull/1362/files).

This change fixes the test and clarifies that it doesn't matter how many entries are returned: our check is only meant to ensure that 3 of them, in any order, contain "Yes".